src/main/java/edu/iris/dmc/station/Application.java

```diff
@@ -58,11 +66,14 @@ public void run(String... args) {
 			String arg = args[i];
 			if ("--verbose".equals(arg) || "-v".equals(arg)) {
 				debug = true;
-				/*
-				 * Logger rootLogger = LogManager.getLogManager().getLogger("");
-				 * rootLogger.setLevel(Level.INFO); for (Handler h : rootLogger.getHandlers()) {
-				 * h.setLevel(Level.INFO); } logger.log(Level.FINEST, "SEED >< XML CONVERTER");
-				 */
+
+				Logger rootLogger = LogManager.getLogManager().getLogger("");
+				rootLogger.setLevel(Level.INFO);
+				for (Handler h : rootLogger.getHandlers()) {
+					h.setLevel(Level.INFO);
+				}
+				logger.log(Level.FINEST, "SEED >< XML CONVERTER");
+
```

Comment:
Can we switch the verbosity level to ERROR or WARN? INFO seems to be too verbose the the loader prints out :

> Mar 17, 2020 3:49:08 PM edu.iris.dmc.station.Application convert
> INFO: Assume file type is SEED
 
every time it is run.

Reply:
This conversion may be resolved by commit 49c41ddd5f9036643484817a78adc789f5aada79. Please review and comment.